a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -658,3 +661,86 @@ def revert(self, result, loadbalancer, amp_data=None, update_dict=None,
         except Exception as e:
             LOG.error('Failed to remove QoS policy: %s from port: %s due '
                       'to error: %s', orig_qos_id, amp_data.vrrp_port_id, e)
+
+
+class HandleVRIDFloatingIP(BaseNetworkTask):
+    """Handle VRID floating IP configurations for member"""
+
+    def __init__(self, *arg, **kwargs):
+        self.fip_port = None
+        super(HandleVRIDFloatingIP, self).__init__(*arg, **kwargs)
+
+    @axapi_client_decorator
+    def execute(self, vthunder, member, vrid):
+        vrid_ip = None
+        if vrid:
+            vrid_ip = vrid.vrid_floating_ip
+
+        floating_ip = a10_utils.get_vrid_floating_ip_for_project(member.project_id)
```

Comment:
May be we rename vrid_ip and floating_ip to db_floating_ip and conf_floating_ip.